spring-cloud-gcp-data-datastore/src/main/java/com/google/cloud/spring/data/datastore/core/mapping/DatastorePersistentProperty.java

```diff
@@ -69,4 +69,11 @@ public interface DatastorePersistentProperty
    * @return {@code true} if the property is lazily-fetched. {@code false} otherwise.
    */
   boolean isLazyLoaded();
+
+  /**
+   * Return whether to skip null value, i.e., skip insertion if value is null.
+   *
+   * @return {@code true} if the null value is skipped. {@code false} otherwise.
+   */
+  boolean isSkipNullValue();
```

Comment:
`DatastorePersistentProperty` is a public interface. Are we confident that it is not used by customer code for customization?

Reply:
Good point, I changed it to `default` method so it is not a breaking change for customers.